src/types/index.ts

```diff
@@ -63,3 +63,26 @@ export interface DandisetStats {
   asset_count: number,
   size: number,
 }
+
+export interface AssetStats {
+  folder: boolean,
+  name: string,
+  created: string|undefined,
+  modified: string|undefined,
+  size: number|undefined,
```

Comment:
```suggestion
  created: string?,
  modified: string?,
  size: number?,
```
This is the convenient syntax for marking something as optional, which is the same as `whatever|undefined`.

Reply:
https://github.com/dandi/dandiarchive/pull/904/commits/77269760e88c29e960c08158ba518fdb104b711d